internal/pkg/cli/interfaces.go

```diff
@@ -218,7 +214,6 @@ type wsSvcReader interface {
 }
 
 type wsPipelineDeleter interface {
```

Comment:
Looks like we can get rid of this interface and use `wsPipelineManifestReader` instead now :)

Reply:
Oh good catch!